MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/kits/basekits/Kit.java

```diff
@@ -122,8 +153,37 @@ public void setDescription(List<String> description) {
 
   public abstract ItemStack getItemStack();
 
-  public abstract void giveKitItems(Player player);
+  public void giveKitItems(Player player) {
+    player.getInventory().clear();
+    kitItems.forEach((item, slots) -> {
+      for (Integer slot : slots) {
+        switch (slot) {
+          case (-5) -> {
```

Comment:
Is there any reason why these cached values are negative-only numbers?

Reply:
Negative numbers are used for armour slots and next available slot as positives would represent player inventory slots.